runtime/src/fees.rs

```diff
@@ -9,12 +9,15 @@ use frame_support::{
     traits::{Currency, ExistenceRequirement, WithdrawReason},
     weights::SimpleDispatchInfo,
 };
-use frame_system::{self as system, ensure_signed};
+use frame_system::{self as system, ensure_root};
+use sp_runtime::traits::EnsureOrigin;
 
 /// The module's configuration trait.
 pub trait Trait: frame_system::Trait + pallet_balances::Trait {
     /// The overarching event type.
     type Event: From<Event<Self>> + Into<<Self as frame_system::Trait>::Event>;
+    /// Required origin for changing fees
+	type FeeChangeOrigin: EnsureOrigin<Self::Origin>;
```

Comment:
so what's the allowed origin? sorry - not super clear here for me as i am reviewing in-between things. maybe a quick call or skip my review.

Reply:
All good, great question. 

This just defines the `FeeChangeOrigin` type to be needed and implement the `EnsureOrigin` trait. Further down in the code, `T::FeeChangeOrigin::try_origin(origin)` is called, which is verifying that the `origin` of the extrinsic satisfied the FeeChangeOrigin. 

In `lib.rs`, we define `type FeeChangeOrigin = pallet_collective::EnsureProportionAtLeast<_1, _2, AccountId, CouncilCollective>;`, where `pallet_collective::EnsureProportionAtLeast<_1, _2, AccountId, CouncilCollective>;` implements the `EnsureOrigin` trait. 

Hope that makes sense – if not, please call me :)